Makefile

```diff
@@ -160,6 +160,12 @@ test/functional:
 	# Run the functional tests against an existing cluster. Make sure you have logged in to the cluster.
 	go clean -testcache && go test -v ./test/functional -timeout=80m
 
+.PHONY: test/products
+test/products:
+	# Running the products tests against an existing cluster. Make sure you have logged in to the cluster.
+	$(CONTAINER_ENGINE) pull quay.io/integreatly/delorean-cli:master
+	$(CONTAINER_ENGINE) run -e KUBECONFIG=/kube.config -v "${HOME}/.kube/config":/kube.config:z -v $(shell pwd)/test-containers.yaml:/test-containers.yaml quay.io/integreatly/delorean-cli:master delorean tests run --test-config ./test-containers.yaml --output ./test-results/ --namespace products-tests
```

Comment:
Add --rm to any "run" commands or you end up with lots of containers left around https://docs.docker.com/engine/reference/run/#clean-up---rm